docusaurus/docs/recipes/contentful.md

```diff
@@ -0,0 +1,514 @@
+---
+id: contentful
+title: Implementing Contentful
+sidebar_label: Contentful
+---
+
+This recipe aims to guide you through the implementation of Contetful in the project and also provides custom solutions for different use-cases.  
+
+### What is a CMS?
+
+A CMS, or Content Management System, is a platform that helps in the creation and management of content to be consumed in a web application. In short, these services provide a database and a database management system with a layer of usability on top that makes them inclusive to users without technical expertise.
+
+### Contentful
+
+[Contenful](https://www.contentful.com/) is one such CMS, and the object of this recipe.
+
+For more information about **Contentful**'s API please [refer to their documentation](https://www.contentful.com/developers/docs/).
+
+## Modeling your schema
+
+As a developer working on the App that will consume the content stored in the CMS, you're also responsible for creating and modeling the database itself. This will mean creating new content models that must be usable by the client team which will eventually handle **Contentful** after the hand off of the project. One **major** consideration to have is the usability and readability of the content models you're creating. Remember that they are meant to be usable without any previous technical knowledge of this context. 
```

Comment:
> modeling the database

Change to "modeling the schema"?

Reply:
I don't have a personal preference, so sure, I'll change it

Reply:
Generally, considering this is a recipe for Contentful, I'd say we should adhere to Contentful's where relevant. I've suggested "schema" instead of "database", but is "schema" the term adopted by Contentful?

Reply:
Looks like Contentful calls it [data model](https://www.contentful.com/developers/docs/concepts/data-model/). But I think schema is understandable from a developer's point of view.